test/unit/tab/index.test.tsx

```diff
@@ -35,143 +37,140 @@ test('adds a class from state.classList', () => {
 });
 
 test('has a foundation after mount', () => {
-  const wrapper = mount(<Tab active />);
+  const wrapper = mount<Tab>(<Tab active />);
   assert.exists(wrapper.instance().foundation_);
 });
 
 test('if props.active updates to true, activate is called with previousIndicatorClientRect prop', () => {
-  const clientRect = {test: 1};
-  const wrapper = shallow(<Tab previousIndicatorClientRect={clientRect}/>);
-  wrapper.instance().activate = td.func();
+  const clientRect = {test: 1} as unknown as ClientRect;
```

Comment:
```typescript
export function coerceForTesting<T>(value: {}): T {
  return value as T;
}
```

I wrote that function for our tests in the Sketch plug-in.  It does the same thing, but in a more obvious way:

```typescript
   const clientRect = coerceForTesting<ClientRect>({test: 1});
```

Reply:
I recommend using it in all the places here you are using `as`.